bip-0009.mediawiki

```diff
@@ -34,18 +44,14 @@ or ''failed''.  Transitions are considered after each
 retarget period.
 
 '''Soft Fork Support'''
-Software which supports the change should begin by setting B in all blocks
-mined until it is resolved.
+Software which supports the change MUST set B in all blocks mined until it is resolved.
 
     if (BState != activated && BState != failed) {
         SetBInBlock();
     }
 
 '''Success: Lock-in Threshold'''
-If bit B is set in 1916 (1512 on testnet) or
-more of the 2016 blocks within a retarget period, it is considered
-''locked-in''.  Miners should continue setting bit B, so uptake is
-visible.
+For each retarget interval, the number of blocks in which bit B is set is counted. If bit B is set in at least some threshold number of blocks (1916 on mainnet, 1512 on testnet) of the 2016 blocks within a retarget period, it is considered ''locked-in''. Miners SHOULD continue setting bit B so uptake is visible.
```

Comment:
Is it MUST? I thought we agreed that until activation it was still considered optional or else lock-in itself would be a soft fork.